config/config.go

```diff
@@ -74,11 +75,13 @@ func GetFilePermissionWarnings(path string) ([]string, error) {
 	}
 
 	var warnings []string
-	if info.Mode()&0040 > 0 {
-		warnings = append(warnings, fmt.Sprintf("Rancher configuration file %s is group-readable. This is insecure.", path))
-	}
-	if info.Mode()&0004 > 0 {
-		warnings = append(warnings, fmt.Sprintf("Rancher configuration file %s is world-readable. This is insecure.", path))
+	if runtime.GOOS != "windows" {
```

Comment:
We should invert this conditional to avoid unneeded nesting 

```go
if runtime.GOOS == "windows" {
   return nil, nil
}
```

Reply:
Love to see it implemented :-), Regards from the lonely windows user of ranchercli